packages/stores/types.ts

```diff
@@ -613,6 +640,11 @@ export type TCoreStores = {
     modules: Record<string, any>;
     notifications: TNotificationStore;
     traders_hub: TTradersHubStore;
+    gtm: Record<string, unknown>;
+    pushwoosh: Record<string, unknown>;
+    contract_replay: Record<string, unknown>;
+    chart_barrier_store: Record<string, unknown>;
+    active_symbols: Record<string, unknown>;
```

Comment:
```suggestion
```
@henry-deriv  I think I commented this before and you removed it 🤔 Maybe it was another PR 🤔

Reply:
removed

Reply:
@farzin-deriv change request needs to be removed